.gitignore

```diff
@@ -0,0 +1,79 @@
+# Temporary Build Files
```

Comment:
I know this `.gitignore` file is generated, but can you remove all entries not generated by the build process of the operator? 

All entries related to IDEs can be in a single global gitignore file on the developer's machine, not related to this repo.

Reply:
Moved to manually add only what we need